pom.xml

```diff
@@ -96,7 +96,11 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-model-compiler</artifactId>
+        <version>${version.org.kie}</version>
```

Comment:
You already import the bom so you don't need to declare again `drools-model-compiler` version

Reply:
Without this dep the Rule* classes aren't generated